markdown/events/2022/bytesize-gitpod.md

```diff
@@ -25,3 +25,121 @@ It is our hope that these talks / videos will build an archive of training mater
+If we just see that the program is running and we'll see all the tasks connected are running through, we can then... I'll just wait for it to finish, shouldn't take too long... This pipeline, I should have mentioned, is actually taking FASTQ reads. It's running it through FASTQC to get quality control of those particular samples and it's using MultiQC to push them all together to make a really nice plot of all the different features and the quality scores of the FASTQ reads.
+
+[11:20](https://www.youtube.com/watch?v=kBoC6QBU-M0t=680)
+This is a quick way of running a pipeline and you can do this for any of the nf-core pipelines that have this gitpod.yaml. It's quite fun to do. It means you can just really test out a new pipeline and see exactly how it works and if it works with your data. You can drop and drag your files into this explorer area to bring files in. There's also other ways that are normally here. There'll be a file in store, I don't know why it's not here. Might be because of the screen. Anyway, you can drop and drag files in here. I'll just take that off just see if we can find it. Yes, here we go. Sorry if I have the actual full screen. You have options here that you can save files to your local machine and you can open files and do all the kind of things you would expect to do in a normal computer.
+
+[12:06](https://www.youtube.com/watch?v=kBoC6QBU-M0t=726)
+The other thing, now it's finished. In our results file we have our results and if we look in MultiQC, for example, we have the MultiQC report. Again, you can use these preview buttons, so that you can render these particular results into a nice file where you can see everything that's going on. Here is the results of that FASTQC result in MultiQC. I think this is quite cool to know that you can you can just quickly run these pipelines.
+
+[12:36](https://www.youtube.com/watch?v=kBoC6QBU-M0t=756)
+Within this particular environment you might want to change something. So maybe you're particularly interested in changing a particular module. You can still change everything in a Gitpod environment and once you change something... I'll just add... I won't actually push this to the branch, because people won't be happy. But if I add on `cat` or whatever I've changed here. If I'm changing the code here you'll notice on the left hand side, on the source control, it's noticed that one of the files within your repository has changed. If we click this button it said that we've changed the main.nf and you can now use the git functions within Gitpod to add that particular change and you can also then make a message for that particular change. Also, down the bottom you can see that we have this development branch - on the bottom left-hand side. If we click this we can also choose to create a new branch. If you've made changes to the code this is a very quick way to create a new branch and show that we can push it back to the repository and then do a pull request.
```

Comment:
```suggestion
Within this particular environment, you might want to change something. So maybe you're particularly interested in changing a particular module. You can still change everything in a Gitpod environment and once you change something... I'll just add... I won't actually push this to the branch, because people won't be happy. But if I add on `cat` or whatever I've changed here. If I'm changing the code here you'll notice on the left-hand side, on the source control, it's noticed that one of the files within your repository has changed. If we click this button it said that we've changed the main.nf and you can now use the git functions within Gitpod to add that particular change and you can also then make a message for that particular change. Also, down the bottom, you can see that we have this development branch - on the bottom left-hand side. If we click this we can also choose to create a new branch. If you've made changes to the code this is a very quick way to create a new branch and show that we can push it back to the repository and then do a pull request.
```